engine/dlib/src/dlib/profile.cpp

```diff
@@ -345,7 +345,7 @@ namespace dmProfile
     {
         if (!g_IsInitialized)
         {
-            dmLogError("dmProfile is not initialized");
+            dmLogOnceError("dmProfile is not initialized");
```

Comment:
Prevent constant spam in release version of the engine

Reply:
I think it's ok to remove the printout, since it's likely to be annoying to have an "error" in the logs.

Reply:
it makes sense.